### PR TITLE
Organisation fn read success testcase

### PR DESCRIPTION
**Pull request by cyberkarthik01**
Organisation fn read success testcase

### CODE REVIEW

File: sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/ReadOrganisationTest.java [hunk 1]
```diff
@@ -62,4 +73,46 @@ public void testReadOrganisationFailure(String testName, HttpStatus httpStatusCo
...
+            getSearchOrgUrl(),
+            REQUEST_JSON,
+            MediaType.APPLICATION_JSON,
+            false,
+            httpStatusCode,
+            RESPONSE_JSON);
+  }
+
+  private void beforeTest() {
+    if(StringUtils.isBlank(orgId)){
```
Comment (bvinayakumar) [inline sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/ReadOrganisationTest.java]: Format code.

File: sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/ReadOrganisationTest.java [hunk 2]
```diff
@@ -62,4 +73,46 @@ public void testReadOrganisationFailure(String testName, HttpStatus httpStatusCo
...
+      getAuthToken(this, true);
+      OrgUtil.createOrg(
+              this,
+              testContext,
+              "templates/organisation/create",
+              "testCreateOrgSuccessWithProviderAndExternalId",
```
Comment (bvinayakumar) [inline sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/ReadOrganisationTest.java]: testCreateRootOrgSuccessWithProviderAndExternalId

Check naming of other such tests.

File: sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/ReadOrganisationTest.java [hunk 3]
```diff
@@ -62,4 +73,46 @@ public void testReadOrganisationFailure(String testName, HttpStatus httpStatusCo
...
+    return new Object[][] {
+            new Object[] {TEST_NAME_READ_ORG_SUCCESS_WITH_VALID_ORG_ID, HttpStatus.OK},
+            new Object[] {TEST_NAME_READ_ORG_SUCCESS_WITH_VALID_PROVIDER_AND_EXTERNAL_ID, HttpStatus.OK}
+    };
+  }
+  @Test(dataProvider = "readOrgSuccessDataProvider")
+  @CitrusParameters({"testName", "httpStatusCode"})
+  @CitrusTest
+  public void testReadOrganisationSuccess(String testName, HttpStatus httpStatusCode) {
+    variable("rootExternalId", externalId);
```
Comment (bvinayakumar) [inline sunbird_service_api_test/src/test/java/org/sunbird/integration/test/org/ReadOrganisationTest.java]: rootOrgExternalId

**Comment by cyberkarthik01** [comment]
@bvinayakumar incorporated all the comments. Please review it once

**Pull request merged**